### PR TITLE
Fix issue 59

### PR DESCRIPTION
**Pull request by Munter**
Don't load node modules for middlewares that are not explicitly requested in the options passed down by the user.

ping @gustavnikolaj @papandreou

### CODE REVIEW

**Comment by gustavnikolaj** [comment]
LGTM

File: lib/createLiveStyleApp.js [hunk 1]
```diff
@@ -52,10 +52,13 @@ module.exports = function createLiveStyleApp(options) {
     }
 
     if (options.processImage) {
+        processImage = require('express-processimage');
+
         app.use(processImage({root: options.root}));
     }
 
     if (options.autoprefixer) {
+      autoprefixer = require('express-autoprefixer')
```
Comment (papandreou) [inline lib/createLiveStyleApp.js]: Indentation, missing semicolon

File: lib/createLiveStyleApp.js [hunk 2]
```diff
@@ -67,10 +70,14 @@ module.exports = function createLiveStyleApp(options) {
 
     if (options.root) {
         if (options.compiless) {
+            compiless = require('express-compiless');
+
             app.use(compiless({root: options.root}));
```
Comment (papandreou) [inline lib/createLiveStyleApp.js]: You could save the var and just go `app.use(require('express-compiless')({root: options.root}));`

File: lib/createLiveStyleApp.js [hunk 3]
```diff
@@ -67,10 +70,14 @@ module.exports = function createLiveStyleApp(options) {
...
         }
 
         if (options.compilesass) {
+            compilesass = require('express-compile-sass');
```
Comment (papandreou) [inline lib/createLiveStyleApp.js]: Same here

File: lib/createLiveStyleApp.js [hunk 4]
```diff
@@ -80,6 +87,8 @@ module.exports = function createLiveStyleApp(options) {
         }
 
         if (options.jsxtransform) {
+            jsxtransform = require('express-jsxtransform');
```
Comment (papandreou) [inline lib/createLiveStyleApp.js]: Same here

**Pull request merged**